cli/src/helpers/local/configFileGlobal.ts

```diff
@@ -1,21 +1,88 @@
 import chalk from 'chalk';
-import fs from 'fs';
+import fs, {FSWatcher, readFileSync, WatchEventType} from 'fs';
 import { getTypedStoatConfig, readStoatConfig } from '../../../../action/src/configHelpers';
 import { StoatConfigSchema } from '../../../../action/src/schemas/stoatConfigSchema';
-import { getTemplate } from '../../../../action/src/templateHelpers';
+import {
+  getRemoteDefaultTemplate,
+  getTemplateFormat
+} from '../../../../action/src/templateHelpers';
 import { Template } from '../../../../action/src/types';
 
-import { findStoatConfigPath } from '../pathHelpers';
+import {findGitRoot, findStoatConfigPath} from '../pathHelpers';
```

Comment:
Nitpick. The code is not auto formatted. `husky` is probably not working for `cli`.

Reply:
Updated to what I think is a better husky configuration. There's no root .husky now, only .husky/pre-commit for subprojects.

Reply:
Nice.